pkg/util/pem.go

```diff
@@ -43,8 +43,8 @@ import (
 // https://www.rfc-editor.org/rfc/rfc7468
 
 // See also https://github.com/golang/go/blob/5d5ed57b134b7a02259ff070864f753c9e601a18/src/crypto/x509/cert_pool.go#L201-L239
-func ValidateAndSanitizePEMBundle(data []byte) ([]byte, error) {
-	certificates, err := ValidateAndSplitPEMBundle(data)
+func ValidateAndSanitizePEMBundle(data []byte, filterExpiredCerts bool) ([]byte, error) {
+	certificates, err := ValidateAndSplitPEMBundle(data, filterExpiredCerts)
```

Comment:
suggestion: I'm uneasy about making a breaking change to an exported function here (and below with `ValidateAndSplitPEMBundle`) - I don't think we need to.

Can we change this so it's not breaking?